addons/vagrant/inventory/group_vars/wireless/hostapd.yml

```diff
@@ -0,0 +1,37 @@
+---
+hostapd__configs:
+  - name: open
+    hostapd__auth_algs: '3'
+    hostapd__auth_server_addr: '{{ packetfence_server_mgmt_ip | default(ansible_all_ipv4_addresses[0]) }}'
+    hostapd__auth_server_port: '1812'
+    hostapd__auth_server_shared_secret: 'anHardSharedSecret'
+    hostapd__ctrl_interface: '/var/run/hostapd'
+    hostapd__ctrl_interface_group: '0'
+    hostapd__driver: 'nl80211'
+    hostapd__hw_mode: 'g'
+    hostapd__interface: 'wlan0'
+    hostapd__logger_syslog: '1'
+    hostapd__macaddr_acl: '2'
+    hostapd__nas_identifier: 'HostAP-PF'
+    hostapd__own_ip_addr: '{{ packetfence_server_mgmt_ip | default(ansible_all_ipv4_addresses[0]) }}'
```

Comment:
```suggestion
    hostapd__own_ip_addr: '{{ ansible_all_ipv4_addresses[0]) }}'
```

Reply:
We certainly need to use ansible_all_ipv4_addresses[1]

Reply:
We should use mgmt IP of wireless01 from inventory

Reply:
I guess we could we use `inventory_hostname` instead of wireless01?